### DISCUSSION TITLE
Please add dbus-serialbattery to paket manager

### DISCUSSION THREAD

**Discussion by tazinblack** [Louisvdw/dbus-serialbattery#506] [Ideas]
Would be very nice to install and update dbus-serialbattery with kwindrem's venus paket manager, which can be installed as shown here:

[https://github.com/kwindrem/SetupHelper
](https://github.com/kwindrem/SetupHelper
)

This would also give non IT people the chance to easy use it.

Thanks for your great work!

**Comment by mr-manuel** [comment]
There is already an issue open https://github.com/Louisvdw/dbus-serialbattery/issues/67

**Reply by tazinblack** [reply]
Oh, my fault.
Sorry, I didn't saw this! 
Thanks a log

**Reply by mr-manuel** [reply]
Please mark it as answer, thanks

**Reply by tazinblack** [reply]
> Please mark it as answer, thanks

I would love to, but I don't know how. Is this only possible for issues?
Sorry, I'm just a github beginner.

And by the way, is the packet manager integration already working or is the work on it still in progress?

**Reply by mr-manuel** [reply]
Oh, it's an Idea and not a Q&A. In this case I think it doesn't work.

Seem like nobody even started to integrate it at the moment.

Even if you can install it with the paket manager, at the moment you still have to change the config file via SSH.
